zipline/pipeline/filters/filter.py

```diff
@@ -488,3 +488,17 @@ def _compute(self, arrays, dates, assets, mask):
                 asset=self._asset, start_date=dates[0], end_date=dates[-1],
             )
         return out
+
+
+class StrictlyTrueFilter(CustomFilter):
+    """
+    A Filter that requires its inputs to have
+    been True for the last `window_length` days.
+
+    **Default Inputs**: None
+
+    **Default Window Length**: None
+    """
+
+    def compute(self, today, assets, out, arg):
+        out[:] = (sum(arg) == self.window_length)
```

Comment:
This is badly incorrect:
```
In [1]: data = np.arange(25).reshape(5, 5)

In [2]: data
Out[2]:
array([[ 0,  1,  2,  3,  4],
       [ 5,  6,  7,  8,  9],
       [10, 11, 12, 13, 14],
       [15, 16, 17, 18, 19],
       [20, 21, 22, 23, 24]])

In [3]: sum(data)
Out[3]: 300
```

If tests are passing here, then we're missing coverage in a serious way.